nvtabular/ops/base.py

```diff
@@ -62,13 +62,12 @@ def transform(self, columns: ColumnNames, df: DataFrameType) -> DataFrameType:
         """
         raise NotImplementedError
 
-    def output_column_names(self, columns: ColumnNames) -> ColumnNames:
-        """Given a set of columns names returns the names of the transformed columns this
-        operator will produce
+    def output_columns(self, columns: Columns) -> Columns:
+        """Given a set of columns names returns transformed columns this operator will produce
 
         Parameters
         -----------
-        columns: list of str, or list of list of str
+        columns: Columns
```

Comment:
This seems like it would be clearer what's happening (to me anyway) if it was `List[Column]`

Reply:
This point is an outgrowth of much earlier decisions in the code base (e.g. `ColumnNames`) so it may or may not be worth addressing here, but I'm still wondering if we can massage the code in a direction where plain un-nested lists would suffice. In order to do that, we'd have to figure out a way to make feature crosses across two lists of columns work with the `Operator` API somehow.